docs/cli/run.md

```diff
@@ -90,6 +90,9 @@ before booting the VM. For example `-copy-in-before path/on/host:/path/on/guest`
 - `-copy-out-after`: Copy the contents of a guest directory to a location on the hose
 after shutting down the VM. For example `-copy-out-after path/on/host:/path/on/guest`
 
+- `-qemu-bin-name NAME`: Use `NAME` instead of `qemu-system-x86_64` as QEMU
+then binary.
```

Comment:
Typo

Reply:
?!?!?!!?!? I had just tried to insert 'the' before pushing. sorry. :(